core/services/chainlink/testdata/config-full.toml

```
@@ -3,6 +3,11 @@ InsecureFastScrypt = true
 RootDir = 'test/root/dir'
 ShutdownGracePeriod = '10s'
 
+[AuditLoggerConfig]
```

I think we can leave off the `Config` suffix from the field name, since it is implied by the context.
```suggestion
[AuditLogger]
```